src/encoded/schemas/file_other.json

```diff
@@ -0,0 +1,41 @@
+{
+    "title": "Other file",
+    "description": "Other files to ingest.",
+    "id": "/profiles/file_other.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["file_format", "award", "lab"],
```

Comment:
I think @aschroed and/or @willronchetti was requesting this not require "award" or "lab". (They could be optional, so I wouldn't change other things, but I think these ontology things, at least, are not specific to at least an award and maybe even a lab. Are those attributes required by `File`?)

Reply:
Required properties are not defined in the schema of abstract items.  However, all other current specific File type items do require Lab and Award - if the idea is that this Other file type could be used for other purposes than Ingestion and you would want to have the usual control of permissions on it then this requirement probably should be maintained - it is the OntologyTerm items themselves that lack this requirement.  However, if the goal is eventual full automation by the system and these FileOther items are to be posted to the portal by the system then either these properties would potentially need to be hard coded or passed in somehow or these properties should not be required.